templates/server/index.html

```diff
@@ -28,34 +28,84 @@
-            </ul>
+        <div class="combined-list">
+        <ul class="list-ticks--canonical six-col">
+            <li>Supported for five years by Canonical</li>
+            <li>
+                Runs on all major architectures &ndash; x86, x86-64, ARM v7,
+                ARM64, POWER8 and IBM s390x (LinuxONE)
+            </li>
+            <li>
+                ZFS stable, feature-rich file system with snapshot
+                capabilities
+            </li>
+            <li>
+                LXD Linux container hypervisor enhancements including QoS and
+                resource controls (CPU, memory, block I/O, storage quota)
+            </li>
+            <li>Install Ubuntu Core Snaps</li>
+        </ul>
+        <ul class="list-ticks--canonical six-col last-col">
+            <li>
+                First production release of DPDK &ndash; line speed kernel
+                networking
+            </li>
+            <li>Linux 4.4 kernel and systemd service manager</li>
+            <li>
+                Certification as a guest on AWS, Microsoft Azure, Joyent, IBM
+                and HP Cloud
+            </li>
+            <li>
+                Updates to Tomcat (v8), Postgresql (v9.5), Docker v(1.10),
+                Puppet (v3.8.5), Qemu (v2.5), Libvirt (v1.3.1), LXC (v2.0),
+                and MySQL (v5.6)
+            </li>
+        </ul>
+    </div>
+        <div class="eleven-col">
+            <p><a class="external"
+            href="https://wiki.ubuntu.com/{{lts_release_name}}/ReleaseNotes">
+                Read more in the release notes
+            </a></p>
         </div>
-        <div class="six-col last-col">
-            <h3>Ubuntu OpenStack</h3>
-            <ul class="list-ticks--canonical">
-                <li>Supported for five years by Canonical</li>
-                <li>Updated to the Mitaka release, including automated installation, queuing/notification and the integration of database-as-a-service</li>
-                <li>Nova LXD driver - deploy OpenStack instances as system containers, dramatically increasing per-node density</li>
-                <li>Juju OpenStack bundle -  automated deployment of OpenStack into LXD system containers</li>
-                <li>ZFS support for LXD OpenStack hosts</li>
-                <li>Certified by Microsoft to host Windows Server 2012 and Windows Server 2008 R2 as guests, under its Server Virtualisation Validation Program (SVVP)</li>
-                <li>OpenStack software, Juju, MAAS, LXD on IBM LinuxONE mainframe</li>
-            </ul>
+    </div>
+</section>
+
+<section class="row strip-light">
+    <div class="strip-inner-wrapper equal-height">
+        <div class="four-col equal-height__item equal-height__align-vertically align-center">
+            <img class="priority-0" src="{{ ASSET_SERVER_URL }}e047e34e-picto-landscape.svg"
```

Comment:
I borrowed the image from a similar section. Though design would like to see an image.